Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IncidentsCheck_Widget_CreationDate/IncidentsCheck_Widget_CreationDate.py

```diff
@@ -51,7 +51,7 @@ def main():
             "data": [
                 0
             ],
-            "name": "N\\A",
+            "name": "2020-01-01",
```

Comment:
Is that intentional?

Reply:
Yes because this is a line chart so the fields expecting to get a data